docs/ha-deployment.md

```diff
@@ -0,0 +1,73 @@
+# Container registry HA deployment
+
+## Prerequisites
+
+- Kubernetes cluster v1.20+
+  - Use existing cluster
+    ```bash
+    export KUBECONFIG=/path/to/kubeconfig
+    ```
+  - Alternatively, spawn some dev cluster, e.g. using [KinD](https://kind.sigs.k8s.io/docs/user/quick-start/)
+    ```bash
+    kind create cluster
+    ```
+- Flux CLI
+  - Installation documentation: https://fluxcd.io/flux/installation/#install-the-flux-cli
+    ```bash
+    curl -s https://fluxcd.io/install.sh | sudo FLUX_VERSION=2.2.3 bash
+    flux install
+    ```
+    
+## Install and wait for operators
+
+```
+$ kubectl apply -k operators/
+$ flux get helmrelease -n default
+NAME                    REVISION        SUSPENDED       READY   MESSAGE
+cert-manager            v1.11.0         False           True    Release reconciliation succeeded
+ingress-nginx           4.5.2           False           True    Release reconciliation succeeded
+postgres-operator       1.9.0           False           True    Release reconciliation succeeded
+redis-operator          3.2.7           False           True    Release reconciliation succeeded
+```
+
+> Note: Install separate operators by e.g.:
+> ```
+> kubectl apply -k operators/redis/
+> kubectl apply -k operators/postgres/
+> ```
+
+## Create redis and postgres clusters
+
+```
+envs/public-ha/redis/redis-secret.bash # pwgen needs to be installed
+kubectl apply -k envs/public-ha/redis/
+kubectl apply -k envs/public-ha/postgres/
+```
+
+## Install Harbor
+
+- Take *ingress-nginx-controller* LoadBalancer IP address and create DNS record for Harbor.
```

Comment:
Please, update `harbor-config.yaml` here and remove `registry.scs.dnation.cloud` url - we can instruct the user to change that in this step

Reply:
fixed d0f77c382cb2216e5fc82a4939666d5810c2ec61